package.json

```
@@ -18,11 +18,13 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "storybook": "./node_modules/.bin/start-storybook -p 9009 -s public"
+    "storybook": "./node_modules/.bin/start-storybook -p 9009 -s public",
+    "lint": "./node_modules/.bin/eslint src"
```

no need to reference this directly, npm will pick it up.
There are also test simulators located in `test` that should be linted too.

```suggestion
    "lint": "eslint src test"
```